Sources/Client/Client/Client+Request.swift

```diff
@@ -338,8 +338,10 @@ extension Client {
             do {
                 let podTrunk = try JSONDecoder().decode(PodTrunk.self, from: data)
                 if let latestVersion = podTrunk.versions.last?.name, latestVersion > Environment.version {
-                    ClientLogger.logger("📢", "", "StreamChat \(latestVersion) is released (you are on \(Environment.version)). "
-                        + "It's recommended to update to the latest version.")
+                    ClientLogger.logger("📢",
+                                        "",
+                                        .info, "StreamChat \(latestVersion) is released (you are on \(Environment.version)). "
+                                            + "It's recommended to update to the latest version.")
```

Comment:
That looks weird :) Can you create a var with log message and rewrite these 4 lines with 2?

Reply:
Why? It only looks weird because we don't have argument labels. We'll overhaul logger soon anyway
(I missed a newline after `.info`, adding it now)